### ISSUE TITLE
Getting a error on WoW SoD - 'ActionBarActionButtonDerivedMixin' (a nil value)

### ISSUE THREAD

**Issue by csBlackWolf** [Choonster-WoW-AddOns/CustomTooltips#7]
Hello. 

When i try to use this Addon (v1.20.0) on World of Warcraft Classic Season of Discovery (v1.15.2-54332 x64) i getting a error.

```
8x .../CustomTooltips_Blizzard/CustomTooltips_Blizzard.lua:15: attempt to index global 'ActionBarActionButtonDerivedMixin' (a nil value)
[string "@CustomTooltips_Blizzard/CustomTooltips_Blizzard.lua"]:15: in main chunk

Locals:
MAX_MACROS = 138
debugprint = <function> defined @CustomTooltips_Blizzard/CustomTooltips_Blizzard.lua:11
ActionBarActionButtonMixin_SetTooltip_Old = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index global 'ActionBarActionButtonDerivedMixin' (a nil value)"
```

I tested it with other addons disabled, except for BugGrabber and BugSack.

**Comment by Choonster** [comment]
I've just released version 1.21.0, which should fix this. I don't have access to WoW Classic, so let me know if any other issues come up.

**Comment by csBlackWolf** [comment]
Thank you for the Update. Sadly i getting a other error now.
```
6x .../CustomTooltips_Blizzard/CustomTooltips_Blizzard.lua:42: hooksecurefunc(): SetTooltip is not a function
[string "=[C]"]: in function `hooksecurefunc'
[string "@CustomTooltips_Blizzard/CustomTooltips_Blizzard.lua"]:42: in main chunk

Locals:
```

**Comment by Choonster** [comment]
Can you try [this alpha version](https://legacy.curseforge.com/wow/addons/custom-tooltips/files/5376013) and see if it works? I've had a look at the UI source code for all of the latest Classic patches and this should work for them.

**Comment by csBlackWolf** [comment]
The alpha version seems to work good. 
There are no lua errors and the tooltip is working like expected. 
![grafik](https://github.com/Choonster-WoW-AddOns/CustomTooltips/assets/30509936/65075e12-94ee-4096-b4c3-f366aab7e5ec)
 
Thank you. 😊

**Comment by Choonster** [comment]
Thanks, I've released this fix as 1.21.1.
